sdk/storage/Azure.Storage.Common/swagger/Generator/src/generator.ts

```diff
@@ -169,11 +170,13 @@ function generateOperation(w: IndentWriter, serviceModel: IServiceModel, group:
     if (sync) {
         w.line(`/// <param name="async">Whether to invoke the operation asynchronously.  The default value is true.</param>`);
     }
+    w.line(`/// <param name="${clientDiagnostics}">The ClientDiagnostics instance used for operation reporting.</param>`);
     w.line(`/// <param name="${operationName}">Operation name.</param>`);
     w.line(`/// <param name="${cancellationName}">Cancellation token.</param>`);
     w.line(`/// <returns>${operation.response.model.description || returnType.replace(/</g, '{').replace(/>/g, '}')}</returns>`);
     w.write(`public static async System.Threading.Tasks.ValueTask<${sendMethodReturnType}> ${methodName}(`);
     w.scope(() => {
+        w.line(`Azure.Core.Pipeline.ClientDiagnostics ${clientDiagnostics},`);
```

Comment:
@tg-msft I hate to make it the first parameter but the way pipeline parameter is done I wasn't able to inject a second one without messing up other logic.